README.rst

```diff
@@ -87,17 +87,21 @@ Building the Driver on Linux and macOS
 Building the Driver on Windows
 ------------------------------
 
-.. |win-vs-version| replace:: VS16 8.0.16 
+.. |win-vs-version| replace:: VS16 
+.. |win-php-version| replace:: 8.1.18 
+
 
 **Note**: Snowflake supports only thread-safe versions of PHP.
 
+You must install Microsoft Visual Studio 2019 (VS16) or earlier with the C++ development installer option.
+
 To build the PHP driver for Windows:
 
 #. Download and install the PHP SDK:
 
-    #. Download PHP 8.0.16 binaries from `<https://windows.php.net/downloads/releases/php-8.0.16-Win32-vs16-x64.zip>`_.
+    #. Download the PHP version binaries from `<https://windows.php.net/downloads/releases/>`_, such as `<https://windows.php.net/downloads/releases/php-8.1.18-Win32-vs16-x64.zip>`_.
```

Comment:
Better to mention here or somewhere that on Windows NTS is not supported, so do not download binaries with "nts" in the package name.

Reply:
I agree with Harry.

Reply:
Done